docs/content/querying/topnmetricspec.md

```diff
@@ -28,39 +28,26 @@ The metric field can also be given as a JSON object. The grammar for dimension v
 |type|this indicates a numeric sort|yes|
 |metric|the actual metric field in which results will be sorted by|yes|
 
-## Lexicographic TopNMetricSpec
+## Dimension TopNMetricSpec
```

Comment:
@vogievetsky, probably not this PR (it's already long) although in general I would be down with adding `ordering` as a synonym for `metric` in topN and adjusting the docs to call them "Orderings" instead of "TopNMetricSpecs".